runtime/lua/vim/lsp.lua

```diff
@@ -1112,19 +1112,45 @@ function lsp.start_client(config)
     end
   end
 
+  ---@private
+  -- Determines whether the given option can be set by `set_defaults`.
+  local function is_empty_or_default(bufnr, option)
+    if vim.bo[bufnr][option] == '' then
+      return true
+    end
+
+    local old_bufnr = vim.fn.bufnr('')
+    local last_set_from = vim.fn.gettext('\n\tLast set from ')
+    local line = vim.fn.gettext(' line ')
+
+    vim.fn.execute('buffer ' .. bufnr)
```

Comment:
Are we able to use `vim.cmd.buffer(bufnr)` instead, here and below?

Reply:
Yes, done. Thanks!